### PR TITLE
gh-97901 add missing text/rtf to mimetypes

### PR DESCRIPTION
**Pull request by amaddio**
As proposed here https://github.com/python/cpython/issues/97901

* Issue: gh-97901

### CODE REVIEW

**Comment by nanjekyejoannah** [comment]
Can we have a test of any form for this, and maybe documentation (am not sure if this is relevant).

**Review by nanjekyejoannah — COMMENTED** [review]
Can we have a test of any form for this, and maybe documentation (am not sure if this is relevant).

**Review by nanjekyejoannah — COMMENTED** [review]
Dont we need any modification in the method `test_guess_all_types` ?

**Comment by amaddio** [comment]
The `test_guess_all_types` test function tests if the `guess_all_types` function is technically functioning (strict, non-strict, no-hits expected, global state). It is not tested if all pre-defined mime type / extension mappings are available.

I guess this is because at the end of they day `types_map` is a pre-seeded list. We would test pythons list behaviour. At this stage we don't assume that feeding a list and checking if the list contains our values is not functioning. 

Nevertheless if you wish I can drop in a line that tests text/rtf explicitly even though I think I break the idea behind the test. Let me know how I should proceed or If there really is another test function/file that I have overlooked that needs to be fixed.

File: Misc/NEWS.d/next/Core and Builtins/2022-10-05-09-33-48.gh-issue-97901.BOLluU.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Mime type text/rtf is now supported by mimetypes.
```
Comment (noamcohen97) [inline Misc/NEWS.d/next/Core and Builtins/2022-10-05-09-33-48.gh-issue-97901.BOLluU.rst]:
```suggestion
Mime type ``text/rtf`` is now supported by :mod:`mimetypes`.
```

**Comment by amaddio** [comment]
Let me know if there is anything more I can do.

**Comment by encukou** [comment]
LGTM! Thank you, and sorry for the delay.

**Pull request merged**